aten/src/ATen/core/boxing/impl/boxing.h

```diff
@@ -236,30 +231,76 @@ struct BoxedKernelWrapper<
 };
 
 //
-// 4. signatures returning a tuple of Tensor references, and taking the same
-// number of Tensor refs as their initial arguments.
+// 4. an out of place op takes one or more non-const Tensor ref arguments at the
+// end of the arg list, and also returns it/them. This specialization supports
```

Comment:
Well, this specialization only supports a single return, no?

Reply:
> Well, this specialization only supports a single return, no?

yup - see the sentence beginning with "This specialization supports..." 😀 

Fair point though, starting the comment this way is confusing, I'll reword